<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiMicrowave</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="S2pTab">
       <attribute name="title">
        <string>S2P</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="MainLayout" stretch="4,1">
          <item>
           <layout class="QVBoxLayout" name="GraphLayout" stretch="2,1">
            <item>
             <layout class="QVBoxLayout" name="WaveformLayout">
              <item>
               <widget class="QGroupBox" name="WaveformBox">
                <property name="title">
                 <string>Graph</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="Waveform"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="LimitLayout">
              <item>
               <widget class="QGroupBox" name="LimitBox">
                <property name="title">
                 <string>Limit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="OperateLayout" stretch="1,1">
            <item>
             <layout class="QVBoxLayout" name="MarkerLayout">
              <item>
               <widget class="QGroupBox" name="MarkerBox">
                <property name="title">
                 <string>Marker</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="FileLayout" stretch="1">
              <item>
               <widget class="QGroupBox" name="FileBox">
                <property name="title">
                 <string>File</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,2,2,0">
                 <item>
                  <widget class="QPushButton" name="ImportButton">
                   <property name="text">
                    <string>Import</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="ExportButton">
                   <property name="text">
                    <string>Export</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="LoadButton">
                   <property name="text">
                    <string>Load</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="StatusBox">
                   <property name="title">
                    <string>Status</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,7">
                    <item>
                     <widget class="QLabel" name="NumLabel">
                      <property name="font">
                       <font>
                        <pointsize>7</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Num of Files:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="Line" name="line">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="FileLabel">
                      <property name="font">
                       <font>
                        <pointsize>7</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Import Files:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="Line" name="line_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="StatusLabel">
                      <property name="font">
                       <font>
                        <pointsize>8</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="scaledContents">
                       <bool>false</bool>
                      </property>
                      <property name="wordWrap">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ExtractTab">
       <attribute name="title">
        <string>Extract</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="OptimizeTab">
       <attribute name="title">
        <string>Optimize</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About HiMicrowave</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>HiMicrowave Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
